bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -71,6 +74,7 @@
     private String callStatusReportPath;
     private String callTimingReportPath;
 
+    private boolean forceJdkTlsProvider = false;
```

Comment:
We can't use SslProvider directly.  In most cases, we want what ever the default configuration that gRPC provides.  When forceJdkTlsProvider is turned on, we have to do more than just override the default SslProvider... the logic is a bit more complex than that.